src/components/Requirements.vue

```diff
@@ -21,7 +21,7 @@
           <div :style="{'border-bottom': minor.display ? `2px solid #${reqGroupColorMap[req.group][0]}` : ''}"
             @click="activateMinor(id)" class="major-title" v-for="(minor, id) in minors" :key="minor.id">
             <p :style="{'font-weight': minor.display ? '500' : '', 'color' : minor.display ? `#${reqGroupColorMap[req.group][0]}` : ''}"  class="minor-title-top">{{minor.minorFN}}</p>
-            <p :style="{'color': minor.display ? `#${reqGroupColorMap[req.group][0]}` : ''}" class="minor-title-bottom">({{user.collegeFN}})</p> <!-- Change for multiple colleges -->
+            <!-- <p :style="{'color': minor.display ? `#${reqGroupColorMap[req.group][0]}` : ''}" class="minor-title-bottom">({{user.collegeFN}})</p> Change for multiple colleges -->
```

Comment:
Super nice. Tho are there minors that you can't take if you're in certain colleges?

Reply:
Yea and according to certain majors too (but we don't have any restrictions yet). i just commented this out bc it was confusing to show the user's school bc it doesn't have any relation with the minor

Reply:
Making an item on Monday for us to add an excludes for minors based on majors 😄